app/lib/actions/katello/organization/simple_content_access/prepare_content_overrides.rb

```diff
@@ -0,0 +1,26 @@
+module Actions
+  module Katello
+    module Organization
+      module SimpleContentAccess
+        class PrepareContentOverrides < Actions::Base
+          def plan(organization_id)
+            @organization = ::Organization.find(organization_id.to_i)
+            @org_name = @organization.name
+            migrator = ::Katello::Util::ContentOverridesMigrator.new(organization: @organization)
+
+            input[:migrator_result] = migrator.execute_non_sca_overrides!
+            plan_self(organization_id: organization_id, organization_name: @org_name)
```

Comment:
This plan_self would look for a run or finalize phase for the task. I don't see any here. Is this needed?